### ISSUE TITLE
Get Open Forest into the USDA Login.gov agreement

### ISSUE THREAD

**Issue by mgwalker** [USDAForestService/USFS-timber-permitting#172]
**Background**

The original question was how to modify the USFS/Login.gov agreement to include identity verification, but we've since learned that USDA is taking an enterprise approach to Login.gov and has now completed a new agreement. That agreement includes identity verification, and USDA is planning to mod the agreement to include USFS before December 1. Now we need to make sure that gets done. 😄

(See #90 for history) 

**Acceptance criteria** 
- [ ] USFS is included in USDA's Login.gov agreement
- [ ] Open Forest can use Login.gov

**Tasks**
- [ ] Find out USDA's timeline for incorporating USFS into its Login.gov agreement
- [ ] Send Open Forest information to Login.gov so they can add it as a USDA application (the USDA Login.gov POC will have to informally approve it before we can use it)
- [ ] Create a new issue to integrate Login.gov into Open Forest when the previous things are done

**Definition of done**
- [ ] Meets acceptance criteria
